rasa/src/Rasa/Internal/Events.hs

```diff
@@ -64,3 +66,8 @@ data Mod
   | Alt
   | Shift
   deriving (Show, Eq)
+
+data BufTextChanged
+  = BufTextChanged Range
```

Comment:
What do you think about including the new text in here too? I think it's a quick win that would be pretty useful!